mmv1/third_party/terraform/services/compute/go/resource_compute_network_firewall_policy_rule_test.go

```diff
@@ -66,6 +66,8 @@ func TestAccComputeNetworkFirewallPolicyRule_update(t *testing.T) {
 }
 
 func TestAccComputeNetworkFirewallPolicyRule_multipleRules(t *testing.T) {
+  // Currently failing
+	acctest.SkipIfVcr(t)
```

Comment:
Why is this failing? Is it already failing on the DCL or is it because of the swap?

Reply:
This test is currently failing because it depends on the NetworkFirewallPolicyAssociation (which is DCL)
Once this PR is merged I'll get the changes and remove this line on the PR for the Association (https://github.com/GoogleCloudPlatform/magic-modules/pull/11366), then the test will run again!

Reply:
Why would changing this resource to not use the DCL cause any sort of change in the relationship between the resources? We should be able to swap the engine underneath this resource without changing any behavior of the resource

Reply:
Hey @slevenick 
I was looking better at the code and running some tests and I finally discovered that the test was failing because it was missing some code.
I don't know why it was missing those resources that I added, but they was causing the error.
Now the test should run.

Reply:
Forgot to mention...
This test is not working on the current **main** branch